src/flownet/network_model/_network_model.py

```diff
@@ -467,6 +513,23 @@ def active_mask(self, model_index) -> List[bool]:
         """
         return (self._grid["model"] == model_index) & (self._grid["ACTNUM"] == 1)
 
+    def bulk_volume_per_flownet_cell_based_on_tube_length(self) -> np.ndarray:
+        """Generate bulk volume per flownet grid cell based on the total length
+        of the active cells in each tube in the FlowNet and the convex hull on
+        the FlowNet network.
+
+        Args:
+            network: FlowNet network instance.
+
+        Returns:
+            A list with bulk volumes for each flownet tube cell.
+        """
+        return (
```

Comment:
Here we don't set the volume in the inactive cells to 0. Should we do it here as well?

Reply:
Now the volume is multiplied with the ACTNUM value (0 or 1), which means that the inactive gridcells are set to volume == zero.